[FIX] typos and minor changes (#426)
zkSNACKs/WasabiDoc: docs/glossary/Glossary.md

:::

:::details
### Hardware Wallet (HWW)

> A hardware wallet is a special type of Bitcoin wallet which generates and stores the user's private keys in a tailor made hardware device.

Reviewer: > stores the user's private keys in a tailor made hardware device.  

This sounds like the device is being manufactured on specific request by the user. I would change it to: 

> stores the user's private keys on a dedicated hardware device.

Author: ```suggestion
A hardware wallet is a special type of Bitcoin wallet which generates and stores the user's private keys on a dedicated hardware device.
```

### Dust

Dust is an UTXO that is uneconomical to spend.

> Also, small portions of bitcoin can lead to serious consequences about one's privacy, for example the so called `forced address reuse attack`.

Reviewer: > serious consequences about one's privacy  

should be changed to: 

> serious consequences for one's privacy

Author: ```suggestion
Also, small portions of bitcoin can lead to serious consequences for one's privacy, for example the so called `forced address reuse attack`.
```

:::details

> ### Wasabika

Reviewer 2: ### Wasabikas

Author: I think the singular is alright.
